api/controller/blog.js

```diff
@@ -6,7 +8,46 @@ exports.getBlog = function(req,res) {
 	Blog.findById(req.params.blog_id,function(err,blog) {
 		if (err)
 			res.send(err);
-		
+
 		res.json(blog);
 	});
-};
\ No newline at end of file
+};
+
+exports.deleteBlog = function(req, res) {
+	Blog.findById(req.params.blog_id, function(err, blog) {
+		if (err) {
+			res.status(500).json(err);
+			return;
+		}
+
+		if (blog != undefined) {
+			apiRoute.AuthChecker(req, res, function() {
```

Comment:
I know , but we supposed to use this api , whenever the user will be logged in . So there is no need to call the authchecker again. if you think from the product side , this api will be called only after the user is logged in. For testing it with postman , we can somehow use other method.